### ISSUE TITLE
Stimela detects wrong cab directory inside virtualenv

### ISSUE THREAD

**Issue by bennahugo** [SpheMakh/Stimela#147]
Trying to reproduce this issue. I have installed the latest PR inside a virtualenv with the default no-system-site-packages behaviour:
```
(pyenv) jenkins@ike:~/jobs/stimela-pr/workspace/24/projects$ which stimela
/var/lib/jenkins/jobs/stimela-pr/workspace/24/projects/pyenv/bin/stimela
(pyenv) jenkins@ike:~/jobs/stimela-pr/workspace/24/projects$ python -c "import stimela ; print stimela.__file__"
/var/lib/jenkins/jobs/stimela-pr/workspace/24/projects/pyenv/local/lib/python2.7/site-packages/stimela/__init__.pyc
```
Yet it detects cabs on another path:
```
.....
docker build --force-rm -f /var/lib/jenkins/jobs/stimela-cron/workspace/58/projects/pyenv/local/lib/python2.7/site-packages/stimela/cargo/cab/ddfacet/tmpcLm74X -t jenkins_cab/ddfacet /var/lib/jenkins/jobs/stimela-cron/workspace/58/projects/pyenv/local/lib/python2.7/site-packages/stimela/cargo/cab/ddfacet
.....
```

**Comment by SpheMakh** [comment]
It was prioritising logged images (from an earlier build). I'm changing default behaviour to prioritise package images.

**Comment by SpheMakh** [comment]
Fixed in #146
